docs/declarative-customization/view-formatting.md

```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
 }
 ```
 
+### Build custom group headers
+
+You can use `groupProps` to format group headers with flexibility to add grouped column's data, display name and item count. You can also add group aggregates in the group headers.
+
+#### Example: Color coded group header
+
+In the example below we have gallery with group headers formatted according to column metadata.

```

Comment:
In the example below we have gallery view with formatted group headers as per column metadata.

Line equivalent space